src/js/config/translations/en.json

```diff
@@ -5,7 +5,10 @@
     "password": "Password",
     "screen": "Login",
     "username": "Username",
-    "loginText": "The currently accepted usernames for this application are:\n\n- standard_user\n- locked_out_user\n- problem_user\n\nAnd the password for all users is:\n\nsecret_sauce",
+    "loginText": {
+      "usernames": "The currently accepted usernames for this application are:\n\n__standard_user__\n__locked_out_user__\n__problem_user__",
+      "password": "And the password for all users is:\n\n__secret_sauce__"
```

Comment:
I see that we've added a custom markup-rendering utility class which converts __ to e.g. bold or underline - it's unfortunate that JSON syntax does not allow comments. I'm concerned that this could be surprising to users.

Reply:
It's like markdown like we used to have with some editors. I understand your concerns, but on the other hand I don't think we are developing an app for best practices for users about how to develop a RN app.